provisioning/roles/mangaki_source/defaults/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+# Installation
+mangaki_source_user: '{{ mangaki_name }}'
+mangaki_source_group: '{{ mangaki_name }}'
+mangaki_source_home: '/home/{{ mangaki_source_user }}'
+mangaki_source_venv_path: '{{ mangaki_source_home }}/venv'
+mangaki_source_settings_path: '{{ mangaki_source_home }}/settings.ini'
+mangaki_source_install_type: 'pypi'
+mangaki_source_package: 'mangaki'
+
+# Django
+mangaki_source_debug: false
+mangaki_source_auth_providers: []
+
+# Database
+mangaki_source_db_host: '127.0.0.1'
+mangaki_source_db_name: '{{ mangaki_db_name }}'
+mangaki_source_db_user: '{{ mangaki_db_user }}'
+
+# Email
+mangaki_source_has_email: true
+mangaki_source_email_host: '127.0.0.1'
+mangaki_source_email_port: 587
+
+# MAL
+mangaki_source_has_mal: false
+
+mangaki_source_has_data: false
```

Comment:
At some point `mangaki_source` was responsible for making the initial migration, and this was used to tell it to do so. The responsability since shifted to `mangaki_sync` and the variable is no longer used; I'll remove it.